po/tr.po

```diff
@@ -236,7 +235,7 @@ msgstr ""
 
 #: src/configcontainer.cpp:189
 msgid "%4i [%6dMB/%6tMB] [%5p %%] [%12K] %-20S %u -> %F"
-msgstr ""
+msgstr "%4i [%6dMB/%6tMB] [  %%%p] [%12K] %-20S %u -> %F"
```

Comment:
I see you and @dennisschagt discussed this in https://github.com/newsboat/newsboat/issues/2441, but wouldn't `[%%%-6p]` be appropriate here? With `[  %%%p]`, different values will have different widths, e.g. `[  %1]` vs. `[  %100]`. The original format keeps the width constant, forming "columns" of text. With `[%%%-6p]`, it would be `[%1     ]` vs. `[%100   ]`, keeping the columns intact.

(This is not a problem for the `%3u %%` format below because that other format doesn't need to align with anything.)

Reply:
Thanks for noticing! I am totally ignorant on these proper shell practices, I'll update shortly!